docs(tutorials/aws): fix grammatical error (#6384)
kubernetes-sigs/external-dns: docs/tutorials/aws.md

## Arguments

> This list is not the full list, but a few arguments that were chosen.

Reviewer: Thanks!
What do you think about the following suggestion?

```suggestion
This is not a complete list, but a few selected items.
```

Author: Applied,thanks
